lib/manageiq/environment.rb

```diff
@@ -62,13 +57,11 @@ def self.install_bundler
       system!("gem install bundler -v '#{bundler_version}' --conservative")
     end
 
-    def self.bundle_install(root = APP_ROOT)
-      system("bundle check", :chdir => root) ||
-        system!("bundle install #{bundle_params}", :chdir => root)
-    end
-
```

Comment:
I thought this was intentional as on a CI system with cached gems, the bundle check prevents upgrading gems.  To be honest I like this better as we will always pull in the latest ensuring we catch problems sooner than later.

Reply:
Aside If you do remove this, `bundle_params` can also be removed.